common/taskana-common/src/main/java/pro/taskana/common/internal/configuration/DB.java

```diff
@@ -29,11 +33,25 @@ public static boolean isPostgres(String dbProductId) {
     return POSTGRES.dbProductId.equals(dbProductId);
   }
 
+  public static boolean isOracle(String dbProductId) {
+    return ORACLE.dbProductId.equals(dbProductId);
+  }
+
+  public static boolean isOracleDb(String dbProductName) {
+    return ORACLE.dbProductName.equals(dbProductName);
+  }
+
+  public static String getDatabaseProductName(Connection connection) throws SQLException {
+    return connection.getMetaData().getDatabaseProductName();
+  }
```

Comment:
I really like this method. A quick search through the codebase shows some places don't use this method. Do you mind "fixing" these?
![Screenshot from 2023-02-03 07-37-07](https://user-images.githubusercontent.com/15628173/216529558-253826e5-d21c-45e4-b541-16871f536b66.png)

Reply:
Done